src/components/Button/Button.tsx

```diff
@@ -204,7 +208,9 @@ export function Button({
         )}
       />
     </span>
-  ) : null;
+  ) : (
+    <span />
```

Comment:
Fixes the button width from changing when the spinner is shown or not.

Reply:
Hey Laura! It looks like this change broke pagination buttons 

![image](https://screenshot.click/screencast_2020-10-06_10-02-28.gif)

Would you be able to find an alternate fix before the next release goes out later this week?

Reply:
The empty span is breaking buttons that are icon only. Noticed on the banner dismissible icons

Reply:
Thanks Dan, Laura fixed this up yesterday [here](https://github.com/Shopify/polaris-react/pull/3392)